Kickstarter-iOS/Features/MessageBanner/Controller/MessageBannerViewController.swift

```diff
@@ -19,7 +20,19 @@ public final class MessageBannerViewController: UIViewController, NibLoading {
   @IBOutlet fileprivate var messageLabel: UILabel!
 
   private var bannerType: MessageBannerType?
-  private var dismissible: Bool = true
+
+  public enum DismissType {
+    // Banner cannot be dismissed and will not be dismissed automatically.
+    case persist
+    // Banner can be dismissed by the user and will be dismissed automatically.
+    // Once the banner is dismissed, the delegate method `messageBannerViewDidHide` is called.
+    case bannerOnly
+    // Banner can be dismissed by the user and will be dismissed automatically.
+    // Once the banner is dismissed, the presenting view controller is popped.
+    case bannerAndParentVC
```

Comment:
minor nit: maybe `bannerToParentVC` is a clearer name for this case?

Reply:
I've renamed this to just `bannerAndViewController` - I think that we were thinking of the "parent vc" differently (in my mind it's the view controller that's getting dismissed, since it's the parent of the message banner). I think the new name is clearer for both of us!